frontend/app/services/personal-information-service.server.ts

```diff
@@ -151,41 +246,44 @@ function createPersonalInformationService() {
   }
 
   function toPersonalInformation(personalInformationApiResponse: PersonalInformationApiResponse): PersonalInfo {
-    const addressList = personalInformationApiResponse.Client.PersonContactInformation?.Address;
+    const addressList = personalInformationApiResponse.BenefitApplication.Applicant?.PersonContactInformation?.Address;
     const homeAddressList = addressList?.filter((address) => address.AddressCategoryCode.ReferenceDataName == 'Home');
     const mailingAddressList = addressList?.filter((address) => address.AddressCategoryCode.ReferenceDataName == 'Mailing');
 
     return {
-      applictantId: personalInformationApiResponse.Client.ClientIdentification ? personalInformationApiResponse.Client.ClientIdentification.filter((clientInfoDto) => clientInfoDto.IdentificationCategoryText === 'Applicant ID').at(0)?.IdentificationID : '',
-      clientId: personalInformationApiResponse.Client.ClientIdentification?.filter((clientInfoDto) => clientInfoDto.IdentificationCategoryText === 'Client Number').at(0)?.IdentificationID,
-      firstName: personalInformationApiResponse.Client.PersonName?.at(0)?.PersonGivenName?.at(0),
-      lastName: personalInformationApiResponse.Client.PersonName?.at(0)?.PersonSurName?.at(0),
-      emailAddress: personalInformationApiResponse.Client.PersonContactInformation?.EmailAddress.EmailAddressID,
-      phoneNumber: personalInformationApiResponse.Client.PersonContactInformation?.TelephoneNumber.FullTelephoneNumber,
+      applicantCategoryCode: personalInformationApiResponse.BenefitApplication.Applicant?.ApplicantCategoryCode.ReferenceDataID,
+      applictantId: personalInformationApiResponse.BenefitApplication.Applicant?.ClientIdentification?.filter((clientInfoDto) => clientInfoDto.IdentificationCategoryText === 'Applicant ID').at(0)?.IdentificationID,
+      clientId: personalInformationApiResponse.BenefitApplication.Applicant?.ClientIdentification?.filter((clientInfoDto) => clientInfoDto.IdentificationCategoryText === 'Client ID').at(0)?.IdentificationID,
+      clientNumber: personalInformationApiResponse.BenefitApplication.Applicant?.ClientIdentification?.filter((clientInfoDto) => clientInfoDto.IdentificationCategoryText == 'Client Number').at(0)?.IdentificationID,
+      birthDate: personalInformationApiResponse.BenefitApplication.Applicant?.PersonBirthDate.dateTime,
+      firstName: personalInformationApiResponse.BenefitApplication.Applicant?.PersonName?.at(0)?.PersonGivenName?.at(0),
+      lastName: personalInformationApiResponse.BenefitApplication.Applicant?.PersonName?.at(0)?.PersonSurName,
+      emailAddress: personalInformationApiResponse.BenefitApplication.Applicant?.PersonContactInformation?.EmailAddress.EmailAddressID,
+      maritalStatus: personalInformationApiResponse.BenefitApplication.Applicant?.PersonMaritalStatus.StatusCode.ReferenceDataID,
```

Comment:
Since the value of the `maritalStatus` isn't actually going to be something like "Single" or "Married" but rather a UUID:

```suggestion
      maritalStatusId: personalInformationApiResponse.BenefitApplication.Applicant?.PersonMaritalStatus.StatusCode.ReferenceDataID,
```

Reply:
Updated: 3489b449c8a8d57369f2afe860d035d0deaf0e1e